util/importers/ldc93s1.py

```diff
@@ -74,6 +75,7 @@ def _compute_source_target(self):
     def _populate_batch_queue(self, session):
         source, source_len, target, target_len = self._compute_source_target()
         while True:
+            time.sleep(1)
```

Comment:
This is the best I have been able to come up so far. I can try to dig deeper, but I fear it will be a bad use of my time.